auth-web/src/models/continuation-review.ts

```diff
@@ -1,8 +1,29 @@
+export enum ContinuationReviewStatus {
+  AWAITING_REVIEW = 'AWAITING_REVIEW',
+  CHANGE_REQUESTED = 'CHANGE_REQUESTED',
+  RESUBMITTED = 'RESUBMITTED',
+  REJECTED = 'REJECTED',
+  ACCEPTED = 'ACCEPTED',
+  ABANDONED = 'ABANDONED'
+}
```

Comment:
As we are planning to extract some of the functionality out of Auth Web and into a new UI, I thought I'd keep this enum along with the "models" below, for easier porting.